administrator-guides/omnichannel-integrations/facebook-integration/README.md

```diff
@@ -0,0 +1,41 @@
+# Facebook Messenger Integration with Rocket.Chat
+
+This integration allows your organization to communicate with Facebook Messenger users within Rocket.Chat.
+
+To set up the integration, request an API key from Rocket.Chat Support. When you receive the key, configure the integration.
+
+## Request an API Key
+
+Send an email to [omni@rocket.chat][1] to request an API key. We process requests as fast as we can, and it may take a few days, but you should have your key and secret soon enough.
+
+## Configure the Integration
+
+* In **Administration**, select **Omnichannel** and enable it (if it is not enabled yet).
+
+* Go to the **Facebook** section and enable the Facebook integration.Enter your _Key_ and _Secret_.
+
+* Click **Save**.
+
+![Facebook section][image-1]
+
+* Return to the main Rocket.Chat interface.
+
+* To see the **LiveChat** settings, click **Administration**.
+
+![Livechat settings][image-2]
```

Comment:
```suggestion
![Omnichannel settings][image-2]
```

You need to update the picture, it's still displaying the `Livechat` term instead of `Omnichannel`.